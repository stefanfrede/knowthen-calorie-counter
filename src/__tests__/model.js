import initModel from '../model';

it('renders correctly', () => {
  const model = initModel;

  expect(model).toMatchSnapshot();
});
