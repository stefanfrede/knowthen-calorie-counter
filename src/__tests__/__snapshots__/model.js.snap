// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Object {
  "calories": 480,
  "description": "Breakfast",
  "editId": null,
  "meals": Array [],
  "nextId": 0,
  "showForm": false,
}
`;
